src/v/cluster_link/model/types.cc

```diff
@@ -115,11 +115,16 @@ update_mirror_topic_properties_cmd::copy() const {
 }
 } // namespace cluster_link::model
 
-auto fmt::formatter<cluster_link::model::mirror_topic_state>::format(
-  cluster_link::model::mirror_topic_state s, format_context& ctx)
-  -> decltype(ctx.out()) {
-    return fmt::format_to(ctx.out(), "{}", to_string_view(s));
-}
+template<typename T>
+struct fmt::formatter<std::optional<T>> : fmt::formatter<string_view> {
+    auto format(const std::optional<T>& v, format_context& ctx) const
+      -> decltype(ctx.out()) {
+        if (!v.has_value()) {
+            return fmt::format_to(ctx.out(), "none");
+        }
+        return fmt::format_to(ctx.out(), "{}", *v);
+    }
+};
```

Comment:
This should be available at `fmt/std.h`?

Reply:
Is it only in newer versions? :( Should we use the version we have defined in `utils/to_string.h` instead?